Server/Services/RatingService.cs

```diff
@@ -75,15 +76,78 @@ private int CalculateScoreDelta(double averageWinningRating, double averageLosin
             return result;
         }
 
-        private Dictionary<Player, GameResult> SplitTeams(IEnumerable<MatchPlayer> matchPlayers, ushort winnerTeam)
+        #endregion
+
+        #region Overthrow
+
+        private Dictionary<string, List<int>> _scores = new Dictionary<string, List<int>>()
         {
-            var result = new Dictionary<Player, GameResult>();
-            foreach (var matchPlayer in matchPlayers)
-                result.Add(matchPlayer.Player, matchPlayer.Team == winnerTeam ? GameResult.Winner : GameResult.Loser);
+            { "forest_solo", new List<int> { 30, 25, 18, 11, 4, -4, -11, -18, -25, -30 } },
+            { "core_quartet", new List<int> { 30, 18, 6, -6, -18, -30 } },
+            { "desert_duo", new List<int> { 30, 15, 0, -15, -30 } },
+            { "temple_quartet", new List<int> { 30, 10, -10, -30 } },
+            { "mines_trio", new List<int> { 30, 0, -30 } },
+            { "desert_quintet", new List<int> { 30, 0, -30 } },
+            { "desert_octet", new List<int> { 30, 0, -30 } }
+        };
+        public Dictionary<ulong, PlayerRatingChange> RecordRankedMatchOverwatch(IEnumerable<MatchPlayer> matchPlayers, string mapName)
+        {
+            var result = new Dictionary<ulong, PlayerRatingChange>();
```

Comment:
`result` is hard to understand without looking up for the context
```suggestion
            var ratingChanges = new Dictionary<ulong, PlayerRatingChange>();
```